mlflow/tracking/client.py

```diff
@@ -713,6 +714,43 @@ def end_span(
 
         span.end()
 
+    def set_trace_tag(self, request_id: str, key: str, value: str):
+        """
+        Set a tag on the trace with the given trace ID.
+
+        The trace can be ongoing one or the one that has already ended and recorded in the backend.
```

Comment:
Does ongoing here mean active?

Reply:
A trace that is started and running i.e. not yet ended.

Reply:
can we use `active` because we use `active` for runs?

Reply:
makes sense!